### ISSUE TITLE
Dragon model legs

### ISSUE THREAD

**Issue by VergilDarkSlayer** [DracoAnimus/Coding#21]
well this is more a question and probaly a suggestion, are the dragon legs supposed to dissapear when in flying mode? because I think it would be better if it was in "crouch" position also the wings are gonna be fixed in the next builds too?

**Comment by wildbill22** [comment]
Yes, it is intentional that the legs disappear when flying. I mentioned that in the release notes. Eventually we would like a tucked in (crouch) position for them, but don't have it yet. Also the wings will have a tucked in position when on the ground eventually, but this hasn't been done yet (currently they are not rendered when on the ground).

**Comment by DSkycroft** [comment]
So,I would like to point out that someone on the MCF who said they were the coder said that he did not want to take the time to improve the models. Is he the real coder and if he is,I don't think he is a very good one since he does not seem interested in improving the mod.

**Comment by wildbill22** [comment]
Yes, that was me on MCF. I think you are taking what I said out of context. See my newest post. We do have a plan for improving the mod, and improving the way the models and rendering of the dragons is on the plan, it just isn't something we are doing right now. Yes I'm a real coder, and I assume you are not, so just maybe I know what I am talking about and you don't? I guess we have now become popular enough to generate our first hate message :) If you want to help, we do need another coder, if you know of one, let us know. We are just two people making a huge mod. This takes a lot of time that we only do in our spare time for fun. We are not paid. Funny how people can complain about something they are getting for free :)

**Comment by wildbill22** [comment]
BTW, VergilDarkSlayer has helped with the mod in several different ways. And we appreciate comments and any help provided.

**Comment by DSkycroft** [comment]
I am sorry,I did take it out of context. I tend to get a little carried away when I think I am being insulted. If you need help with models,I would recommend asking the modeler from the "Kingdom of Craftzoic" mod,he is pretty good at modeling.
